fvm/bootstrap.go

```diff
@@ -76,13 +76,7 @@ type BootstrapParams struct {
 	minimumStorageReservation        cadence.UFix64
 	storagePerFlow                   cadence.UFix64
 	restrictedAccountCreationEnabled cadence.Bool
-
-	// `setupEVMEnabled` == true && `evmAbiOnly` == true will enable the ABI-only EVM
-	// `setupEVMEnabled` == true && `evmAbiOnly` == false will enable the full EVM functionality
-	// `setupEVMEnabled` == false will disable EVM
-	// This will allow to quickly disable the ABI-only EVM, in case there's a bug or something.
-	setupEVMEnabled cadence.Bool
-	evmAbiOnly      cadence.Bool
+	setupEVMEnabled                  cadence.Bool
```

Comment:
This one can be removed too, but that can be done in a different PR.

Reply:
@janezpodhostnik Removed in https://github.com/onflow/flow-go/pull/5587/commits/d95792c96b18f49ec46a65629c2128351a16ec39

Reply:
That actually needs a more changes to work, as I think the new account for storage does not have keys. I'll leave this for another PR :pray: